_third_party/github.com/StackExchange/httpunit/httpunit.go

```diff
@@ -393,6 +393,7 @@ type TestResult struct {
 	GotText     bool
 	GotRegex    bool
 	InvalidCert bool
+	TimeTotal   time.Duration
```

Comment:
I merged over my changes to httpunit, I did not merge back the changes from httpunit here yet (some timout improvements): https://github.com/StackExchange/httpunit/pull/5